policykit/policyengine/models.py

```diff
@@ -1026,6 +1027,143 @@ def __init__(self, *args, **kwargs):
         self.label_suffix = ''
 
 
+class CustomAction(models.Model):
```

Comment:
One thought - right now we still have the idea of Policy Templates via the Policy model (which has an is_template field). I wonder if we want to get rid of that and instead have a new PolicyTemplate class that can bundle an Action/CustomAction + a Procedure + an Execution. This would be useful in the case of say, being able to select from a set of templates (eg "Community vote on OC expenses on Slack" like how Nick is doing). This would fill everything in so the user doesn't have to go through the 3-stage process unless they want to edit something.

Reply:
I like the idea! Right now, if we have already decomposed a policy template into three modules, the policy templates by the field`is_template` in the `Policy` seems repetitive; and we still would like to provide users with some complete policies

Reply:
Yes, and a PolicyTemplate could include multiple CustomActions/Actions and multiple Executions as well (I think in the UI too, we want the ability to allow users to select multiple).